### ISSUE TITLE
기존에 기록한 모듈의 코드를 보는 공간에서,

### ISSUE THREAD

**Issue by bjrambo** [kijin/xe-errorlogger#1]
기존에 오류를 나타낸 공간에서 코드를 수정 한 뒤, 같은 위치의 코드가 수정해야하는 것으로 나타납니다.

8월 27일 8시 47분에 기록된, 오류에서 코드 보기부분에

``` php
if(!$config->~~)
{
$config->~~ = 'Y';
}
```

에서 `$config->~~ = 'Y';`

부분이 오류나서 그 부분을 수정하였습니다.

``` php
if(!$config)
{
$config = new stdClass();
}
```

그래서 오류는 해결되었는데 문제는 공교롭게도, 기존의 입력된 디비내에서는 `$config = new stdClass();`부분이 같은 줄으로 바뀌어 해당 코드가 오류가 있다고 나타나고 있습니다.

(코드를 보여주는 곳이 현재의 코드로 다시 보여주는 형태라 그런건가 봅니다..)

아무래도 이 부분의 저장이 안되어서 그런지 오히려 해당부분이 디버깅을 진행한 부분인지 아닌지 모르는 상황이 발생되는 것 같습니다.

**Comment by kijin** [comment]
네, 현재는 에러에 대한 최소한의 정보만 기록하고, 나머지 정보는 조회 시점에 채워넣고 있습니다. 에러 발생 시점의 backtrace, 변수값들, 원본 소스까지 기록하면 좋겠지만... 매번 그렇게 하자니 DB 용량이나 페이지 로딩 속도가 부담돼서요. (시험삼아 `E_NOTICE`를 켜봤다가 3000개의 에러폭탄을 맞는 장면을 보셨어야 하는데 ㅋㅋㅋ)

DB보다는 임시파일이나 캐싱기능을 사용해서 에러를 기록하도록 하거나, 별도로 저장하지 않고 즉시 화면에 표시하도록 하는 기능도 고려하고 있습니다. 혹시 Nette 프레임워크에서 제공하는 [Tracy](https://tracy.nette.org/)라는 에러처리 기능을 구경해 보셨나요? 이걸 벤치마킹하는 중인데, 그야말로 끝내줘요... ^O^

**Comment by bjrambo** [comment]
오홍 저것도 괜찮겠네요 ㅋㄷㅋㄷ
